server/routes/induction/common/inPrisonWorkController.ts

```diff
@@ -35,8 +35,9 @@ export default abstract class InPrisonWorkController extends InductionController
 
 const toInPrisonWorkForm = (inductionDto: InductionDto): InPrisonWorkForm => {
   return {
-    inPrisonWork: inductionDto.inPrisonInterests.inPrisonWorkInterests.map(workInterest => workInterest.workType),
-    inPrisonWorkOther: inductionDto.inPrisonInterests.inPrisonWorkInterests.find(
+    inPrisonWork:
+      inductionDto.inPrisonInterests?.inPrisonWorkInterests.map(workInterest => workInterest.workType) || [],
```

Comment:
Unrelated to my change, I've added `?` null checks when populating the form objects.